src/ChainRulesTestUtils.jl

```diff
@@ -13,6 +13,7 @@ import FiniteDifferences: rand_tangent
 
 const _fdm = central_fdm(5, 1; max_range=1e-2)
 const TEST_INFERRED = Ref(true)
+const TRANSFORMS_TO_ALT_TEST_TANGENTS = Ref{Vector{Function}}([])
```

Comment:
Why is the `Ref` here?
we should be ok with just `Vector{Function}` since if we want to reset it we can use `empty!` and `append!`/`push!` etc

Reply:
Also shouldn't it be `TRANSFORMS_TO_TEST_ALT_TANGENTS`  ?
we are testing alternatative tangents, not alternatively testing tangents.

Reply:
Oh, interesting, I was using 'test' as a noun, not a verb. I.e. these are the 'transforms to alternative test tangents'. perhaps we can just leave out `TEST_`